pages/common/holehe.md

```diff
@@ -0,0 +1,12 @@
+# holehe
+
+> Holehe checks if an email is attached to an account on sites like Twitter, Instagram, Imgur and over 120 others.
+> More information: <https://github.com/megadose/holehe#-cli-example>.
+
+- Show status across all supported websites for the specified email address:
+
+`holehe {{email}}`
```

Comment:
should we use a placeholder similar to the `gpg` page such as `username@example.com`

Reply:
I personally have no preference here. I guess for now I can default to imitating gpg, though. :thinking: